pandas/core/sorting.py

```diff
@@ -264,7 +264,7 @@ def nargsort(items, kind="quicksort", ascending: bool = True, na_position="last"
     elif na_position == "first":
         indexer = np.concatenate([nan_idx, indexer])
     else:
-        raise ValueError("invalid na_position: {!r}".format(na_position))
+        raise ValueError(f"invalid na_position: {repr(na_position)}")
```

Comment:
you can remove the repr on both of these as its unecessary

Reply:
Thanks for the feedback @jreback, I just pushed with `repr()` removed.